dist/common/scripts/scylla_util.py

```diff
@@ -519,3 +515,12 @@ def set(self, key, val):
     def commit(self):
         with open(self._filename, 'w') as f:
             f.write(self._data)
+
+def get_product(dir):
+    if dir is None:
+        dir = etcdir()
+    try:
+        with open(os.path.join(dir, 'SCYLLA-PRODUCT-FILE')) as f:
+            return f.read().strip()
+    except FileNotFoundError:
+        return 'scylla'
```

Comment:
How is this related to CodeQL/PyLens?

Reply:
Both of them suggested that `from scylla_product import PRODUCT` - and they were right - from pure code base perspective. But then Scylla was installed, install.sh created the scylla_product *Python* file, where this import of PRODUCT took place...